docs/mediapipe.md

```diff
@@ -262,3 +265,18 @@ in the conditions:default section of the deps property:
 - Updates in subconfig files and mediapipe graph files do not trigger model server config reloads. The reload of the full config, including subconfig and graphs, can be initiated by an updated in the main config json file or using the REST API `config/reload` endpoint. 
 
 
+## Known issues <a name="known-issues"></a>
+- MediaPipe `SyncSetInputStreamHandler` options are not validated during graph validation, but graph creation:
+```
+input_stream_handler {
```

Comment:
I think it would be good to include input streams config as well so user knows what this tag_index refers to.

Reply:
done